Fix for right-hand-side drawer, and adding toggle button

Yuripetusko opened the pull request:
fixes #2 
fixes #5

Codepen here

http://codepen.io/Yuripetusko/pen/VYqRNg

Yuripetusko committed:
fixing drawer not dragginh from right

Yuripetusko committed:
Better animation

Yuripetusko committed:
Better animation

Yuripetusko committed:
adding close-fn

Yuripetusko committed:
Updating styles for right-hand-side opened drawer

Yuripetusko committed:
Making toggle function globally available

Yuripetusko committed:
Fixing stuff

Yuripetusko committed:
Various fixes including right-hand-side drawer fix

Yuripetusko committed:
Merge branch 'master' of https://github.com/driftyco/ionic-ion-drawer

Conflicts:
	ionic.contrib.drawer.css

Yuripetusko commented:
Demo: http://codepen.io/Yuripetusko/pen/VYqRNg

Yuripetusko committed:
Fixing merge conflict

Yuripetusko committed:
drag 20% of drawer's width feels better before deciding whether to close it or open it

Yuripetusko committed:
Only freeze scrolling of other elements if we are dragging drawer horizontally

Yuripetusko committed:
same as above

Yuripetusko committed:
Only freeze scroll if dragged above threshold

Yuripetusko commented:
@mlynch any chance this can be reviewed and merged soon, I would like to point my bower to main repo rather than my personal one.

Thanks

EAndreyF commented:
Could you add some documentation about your directive?
Also, if I have left and right menu, how can I toggle it?
And, how to hide menu by tapping on content, not just by slide to left?
Finally, you have error in line 175

Yuripetusko commented:
Thanks
1) It's not my directive, I am just pushing some fixes to it. But I will try to write a bisc docs
2) You can toggle it by changing html attribute on directive. check my plunkr
3) to hide by tapping on content is not implemented atm, but can be done easily
4) I don't see any errors on line 175..?

Yuripetusko commented:
@EAndreyF  are you sure you are looking at my code? https://github.com/Yuripetusko/ionic-ion-drawer/blob/master/ionic.contrib.drawer.js

Aknilam committed:
fix drawer on left position

When you click on opened drawer on left position, it `closes` automatically - moves behind left area.

Yuripetusko committed:
Merge pull request #1 from Aknilam/patch-1

fix drawer on left position
